index.html

```diff
@@ -90,6 +121,48 @@
 
 	<script>
 
+	$(document).ready(function() {
```

Comment:
You can append `?w=1` to the url of any diff view in github to make it ignore whitespace changes.